LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/ExceptionFilterUtility.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.NetworkServer
```

Comment:
Nit should this utility class be shared with other project? If yes, probably in LORaWan?

Reply:
Valid point. I think we should do that as soon as it's actually used from somewhere else. If we don't need it, I would suggest to keep the scope of such utilities as small as possible.